Sources/ArcGISToolkit/Components/Augmented Reality/Utilities/ARSwiftUIView.swift

```diff
@@ -134,22 +135,41 @@ class ARSwiftUIViewProxy: NSObject, ARSessionProviding {
     @objc dynamic var session: ARSession {
         arView.session
     }
-    
-    /// Creates a raycast query that originates from a point on the view, aligned with the center of the camera's field of view.
-    /// - Parameters:
-    ///   - point: The point on the view to extend the raycast from.
-    ///   - target: The type of surface the raycast can interact with.
-    ///   - alignment: The target's alignment with respect to gravity.
-    /// - Returns: An `ARRaycastQuery`.
-    func raycastQuery(
```

Comment:
This method was only used by the `raycast(from:)` method below, so I condensed it away.